substrate/frame/system/src/lib.rs

```diff
@@ -2347,6 +2389,72 @@ impl<T: Config> Lookup for ChainContext<T> {
 	}
 }
 
+/// Hooks for the [`Pallet::run_to_block_with`] function.
+#[cfg(any(feature = "std", feature = "runtime-benchmarks", test))]
+pub struct RunToBlockHooks<'a, T>
+where
+	T: 'a + Config,
+{
+	before_initialize: Box<dyn 'a + FnMut(BlockNumberFor<T>)>,
+	after_initialize: Box<dyn 'a + FnMut(BlockNumberFor<T>)>,
+	before_finalize: Box<dyn 'a + FnMut(BlockNumberFor<T>)>,
+	after_finalize: Box<dyn 'a + FnMut(BlockNumberFor<T>)>,
+	_marker: PhantomData<&'a T>,
+}
```

Comment:
```suggestion
}

```

Reply:
I like grouping the struct and its implementations. (also a lot similar grouping can be found in this lib.rs)

But if you insist, you can commit the suggestion. :)

Reply:
I mean you can group them, I just want to have an empty line in between :P Looks better :D